.devcontainer/Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN apt-get update \
     && apt-get upgrade -y \
     && apt-get install -y --no-install-recommends \
         libssh-dev \
+        shellcheck \
```

Comment:
We do lose control of the version this way, which is probably fine, but just calling it out.  The alternative is that you pull it from the shellcheck like `FROM {shellcheck image} AS shellcheck` and then later a `COPY --from shellcheck` command.